src/components/Proposal/View/Budget/RequestedBudgetCard.tsx

```diff
@@ -19,8 +19,47 @@ interface Props {
   budget: BudgetWithContestants
 }
 
+function requestedAndTotalItems(requestedBudget: number, totalCategoryBudget: number) {
```

Comment:
Shouldn't this be `requestedAndRemainingItems`?

Reply:
When grants are finished (passed, rejected, enacted, out of budget) we'll always display requested over total budget for the category